config/static-site-live-server.js

```diff
@@ -161,6 +163,9 @@ class LiveServer {
       };
 
       const inject = (stream) => {
+        if (fileExt == ".wasm") {
+          res.setHeader('Content-Type', 'application/wasm');
```

Comment:
I think you can revert the changes to this file now.

Reply:
Sure you don't want to keep it? It could make others' lives easier if they get hung up trying to build a new codec. It was something of a vertical climb to figure out how to get it to work (though maybe easier now that the examples are standalone WASM).

Reply:
Okay, sure let's keep it.